xc/xc7/yosys/synth.tcl

```diff
@@ -23,7 +23,7 @@ if { $::env(USE_ROI) == "TRUE" } {
     hierarchy -check -auto-top
 
     # Start flow after library reading
-    synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp -iopad -run prepare:check
+    synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp -iopad -nowidelut -run prepare:check
```

Comment:
`-iopad` is not strictly required anymore. It is still supported to be backwards compatible I guess, but the iopads are enabled by default unless there is `-noiopad`

Reply:
I'm fine with this kind of redunancy.